gulpfile.js

```diff
@@ -87,6 +97,26 @@ gulp.task('babel', function() {
   }
 });
 
+/**
+ * Process unit test files with Babel.
+ */
+gulp.task('babel-tests-unit', function() {
```

Comment:
Why do you call the tasks `tests-unit` in this file? Is it because the integration is called `test-integration`?